mono/mini/mini-amd64-gsharedvt.c

```diff
@@ -148,11 +149,13 @@ get_arg_slots (ArgInfo *ainfo, int **out_slots, gboolean is_source_argument)
 		src [0] = map_reg (sreg);
 		break;
 	case ArgValuetypeInReg:
+	case ArgValuetypeAddrInIReg:
 		nsrc = ainfo->nregs;
 		src = g_malloc (nsrc * sizeof (int));
 		for (i = 0; i < ainfo->nregs; ++i)
 			src [i] = map_reg (ainfo->pair_regs [i]);
 		break;
+	case ArgValuetypeAddrOnStack:
 	case ArgOnStack:
```

Comment:
In ArgValuetypeAddrOnStack ainfo->arg_size will always be 8 so I reused the case to reduce code duplication. I can split into a separate case with nsrc set to 1 if that makes the intent clearer, but with some code duplication.